docs/source/snippets/animovement-tip.md

```diff
@@ -0,0 +1,7 @@
+:::{admonition} Tip for R users
```

Comment:
Use `:::{tip}` ?

Reply:
If I do that, the custom "title"  I want to give is not recognised as a "title". 
So I've found this workaround with admonition, which allows me a custom header. Do you know of another way?

Reply:
Ah thanks, didn't know this!